_posts/2021-04-23-postage.markdown

```diff
@@ -12,10 +12,10 @@ Here's a list of couriers that we recommend using, with a list of pros and cons
 
 | Courier | Pros | Cons | Website |
 | --- | --- | --- | --- |
-| Royal Mail (1-3 days, **recommended for smaller items**) | Cheap for smaller items, can drop off at post offices etc. | Can get costly for larger items, no collection service offered | [Link](https://royalmail.com) |
-| ParcelForce (1-2 days) | 40% off student discount via UNiDAYS, can either drop off at post office/depot or have parcel collected | Without discount, is relatively expensive | [With UNiDAYS](https://parcelforce.com/unidays)/[Without](https://parcelforce.com) |
-| UPS (1-2 days, **recommended for larger items**) | Reasonably priced, can drop off at local UPS access point or arrange collection, can have multiple parcels collected at once for relatively low cost | Insurance can be costly depending on the value of the item | [Link](https://www.ups.com/ship/guided) (make sure to book with UPS.com and not UPSToday) |
-| Hermes (1-5 days) | Cheap for smaller items, drop off at Hermes ParcelShop or collection available | UPS/Parcelforce often cheaper for larger items, can be slow and items frequently go missing | [Link](https://www.myhermes.co.uk/) |
+| Royal Mail (1-3 days, **recommended for smaller items**) | Cheap for smaller items, can drop off at post offices, collection service offered for 72p per item | Costly for larger items (>2kg) | [Link](https://royalmail.com) |
+| ParcelForce (1-2 days) | Can either drop off at post office/depot or have parcel collected | Is relatively expensive | [Link](https://parcelforce.com) |
+| UPS (1-2 days, **recommended for larger items**) | Reasonably priced, can drop off at local UPS access point or arrange collection, can have multiple parcels collected at once for relatively low cost (£2.40) | Insurance can be costly depending on the value of the item | [Link](https://www.ups.com/ship/guided) (make sure to book with UPS.com directly and not UPSToday) |
+| Hermes (1-5 days) | Cheap for smaller items, drop off at Hermes ParcelShop or collection available | NOT RECOMMENDED - UPS/Parcelforce often cheaper for larger items, can be slow and items frequently go missing | [Link](https://www.myhermes.co.uk/) |
```

Comment:
Try to avoid using all-caps for emphasis, instead using bold text instead:

```suggestion
| Hermes (1-5 days) | Cheap for smaller items, drop off at Hermes ParcelShop or collection available | **Not recommended** - UPS/Parcelforce often cheaper for larger items, can be slow and items frequently go missing | [Link](https://www.myhermes.co.uk/) |
```